lib/sequelize.js

```diff
@@ -199,6 +199,8 @@ module.exports = (function() {
     this.importCache = {};
 
     Sequelize.runHooks('afterInit', this);
+
+    Promise.prototype.sequelize = this;
```

Comment:
Other than the fact that the user can't change the namespace - Do we know if this will actually conflict? I.e. will having two seperate sequelize instances actually mess up the context values inside each? I doubt it.